lib/workers/repository/process/extract-update.ts

```diff
@@ -9,16 +9,17 @@ import { PackageFile } from '../../../manager/common';
 import { RenovateConfig } from '../../../config';
 import { BranchConfig } from '../../common';
 
-export type ExtractAndUpdateResult = {
-  res: WriteUpdateResult | undefined;
+export type ExtractResult = {
   branches: BranchConfig[];
   branchList: string[];
   packageFiles?: Record<string, PackageFile[]>;
```

Comment:
What is the impact of making this mandatory?

Reply:
Sorry, I'm not fully understand what do you mean. That was mandatory before my changes.

Reply:
Sorry, *not* mandatory.